aten/src/ATen/cpu/FlushDenormal.cpp

```diff
@@ -24,8 +24,12 @@ bool set_flush_denormal(bool on) {
     _mm_setcsr(csr);
     return true;
   }
-#endif
   return false;
 }
+#else
+bool set_flush_denormal(bool on) {
+  return false;
```

Comment:
Do we want to add a `TORCH_WARN_ONCE()` here as we drop the user flag completely :D

Reply:
Good point, looks like we simply need to add implementation for aarch64 and throw warning for other arches, such as PowerPC, filed https://github.com/pytorch/pytorch/issues/62983